stacs/clms_catalog.json

```diff
@@ -7,27 +7,31 @@
   "links": [
     {
       "rel": "root",
-      "href": "https://github.com/eea/clms-stac/stacs/clms_catalog.json",
-      "type": "application/json",
-      "title": "CLMS Catalog"
+      "href": "./clms_catalog.json",
```

Comment:
General question 
What's the good practice for dealing with urls here ? absolute & relative  ? 
For API, perhaps relative should work fine, but static seems to need actual urls?

Reply:
for the catalog itself I'd put an absolute path. The other I'd put relative paths to show the relationship.